ILEditing/MechanicILChanges.cs

```diff
@@ -835,16 +858,29 @@ private static void ChangeWaterQuadColors(ILContext il)
             cursor.Emit(OpCodes.Ldloc, 8);
             cursor.Emit(OpCodes.Ldloc, 3);
             cursor.Emit(OpCodes.Ldloc, 4);
+            
+            // Caching these values can save a LOT of overhead at runtime.
+            ModWaterStyle sunkenSeaWater = ModContent.Find<ModWaterStyle>("CalamityMod/SunkenSeaWater");
+            ModWaterStyle sulphuricWater = ModContent.Find<ModWaterStyle>("CalamityMod/SulphuricWater");
+            ModWaterStyle sulphuricDepthsWater = ModContent.Find<ModWaterStyle>("CalamityMod/SulphuricDepthsWater");
+            ModWaterStyle upperAbyssWater = ModContent.Find<ModWaterStyle>("CalamityMod/UpperAbyssWater");
+            ModWaterStyle middleAbyssWater = ModContent.Find<ModWaterStyle>("CalamityMod/MiddleAbyssWater");
+            ModWaterStyle voidWater = ModContent.Find<ModWaterStyle>("CalamityMod/VoidWater");
```

Comment:
ModContent.GetInstance calls are preferred over ModContent.Find calls here.

Reply:
Edits made, thanks for the suggestion!